libs/components/popovers/src/lib/modules/dropdown/dropdown-menu.component.ts

```diff
@@ -40,6 +40,7 @@ export class SkyDropdownMenuComponent implements AfterContentInit, OnDestroy {
    * Specifies the HTML element ID of the element that labels
    * the dropdown menu. This sets the dropdown menu's `aria-labelledby` attribute
    * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * For more information about the `aria-labelledBy` attribute, see the [WAI-ARIA Definitions of States and Properties](https://www.w3.org/TR/wai-aria/#aria-labelledBy).
    */
   @Input()
```

Comment:
In the SkyDropdownMenuComponent's `ariaRole` property, I think we also need to update the link that currently points to https://www.w3.org/WAI/PF/aria (redirected from https://www.w3.org/WAI/PF/aria/roles) to point to https://www.w3.org/WAI/PF/aria/#roles. ... Also, I think we need to make the same edit to the SkyDropdownItemComponent`s `ariaRole` property.

Also, the SkyDropdownComponent's `label` property has ARIA repercussions. Do we need to make any updates to that property description?

Reply:
Updated